src/services/mongodb/index.ts

```diff
@@ -38,7 +38,7 @@ export const setupMongoConnection = async () => {
       useFindAndModify: false,
     })
   } catch (err) {
-    baseLogger.fatal({ err, path }, `error connecting to mongodb`)
+    baseLogger.fatal({ err, user, address, db }, `error connecting to mongodb`)
```

Comment:
Is this just so that we don't expose the password in the logs?

Reply:
Yes, it is giving problems with the "GitGuardian Security Checks"